### PR TITLE
Add script to install required packages in Debian

### PR DESCRIPTION
**Pull request by neofob**
contrib/debian/dev-packages.lst:
* List of required packages for Debian development environment

scripts/install-debian-pkgs.sh:
* A simple bash script instaling the required Debian packages

Signed-off-by: Tuan T. Pham <tuan@vt.edu>

### CODE REVIEW

File: scripts/install-debian-pkgs.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (avagin) [inline scripts/install-debian-pkgs.sh]: bash isn't installed in Debian by default

**Comment by avagin** [comment]
Pull requests are not accepted for this project. You need to send all changes into the criu mailing list.
https://criu.org/How_to_submit_patches

**Comment by 0x7f454c46** [comment]
I'm not sure, I've got the idea of the patch.
Package dependencies should be installed with your package manager (apt-get/aptitude in debian).
Why should we have debian-specific shell script for it?
i.e., for arch-linux specifics, this looks like: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=criu
Which is all distribution-specific job.

**Comment by neofob** [comment]
@0x7f454c46 : The patch is for people who wants to compile CRIU from source code in Debian environment.  See the [Installation][0] webpage on CRIU website.

[0]: https://criu.org/Installation

**Comment by neofob** [comment]
Of course, people who want to have different cases for different distros could add more scripts for those cases. I focus on Debian which is what I use for now.

**Comment by neofob** [comment]
@xemul merged this with commit 29d512c435c. Closing this.

**Pull request closed without merging**